vars/getJobTimeouts.groovy

```diff
@@ -2,17 +2,24 @@
 
 List<Integer> call(Map params, String region){
     // handle params which can be a json list
-    def test_config = groovy.json.JsonOutput.toJson(params.test_config)
-    def cmd = """
-    #!/bin/bash
-    export SCT_CLUSTER_BACKEND="${params.backend}"
-    export SCT_CONFIG_FILES=${test_config}
-    ./docker/env/hydra.sh output-conf -b "${params.backend}"
-    """
-    def testData = sh(script: cmd, returnStdout: true).trim()
-    println(testData)
-    testData = testData =~ /test_duration: (\d+)/
-    testDuration = testData[0][1].toInteger()
+    if (params.stress_duration == "") {
+        def test_config = groovy.json.JsonOutput.toJson(params.test_config)
+        def cmd = """
+        #!/bin/bash
+        export SCT_CLUSTER_BACKEND="${params.backend}"
+        export SCT_CONFIG_FILES=${test_config}
+        ./docker/env/hydra.sh output-conf -b "${params.backend}"
+        """
+        def testData = sh(script: cmd, returnStdout: true).trim()
+        println(testData)
+        testData = testData =~ /test_duration: (\d+)/
+        testDuration = testData[0][1].toInteger()
+    } else {
+        stressDuration = params.stress_duration.toInteger()
+        prepareDuration = params.prepare_duration.toInteger()
```

Comment:
what will happen if these parameters don't exist, or are set to anything that not integer? shouldn't we consider it? something like:

```suggestion
        stressDuration = params.stress_duration.toInteger() ?: 0
        prepareDuration = params.prepare_duration.toInteger() ?: 0
```

Reply:
in case it does not exist, running this else condition should not happen at all.

Reply:
@soyacz is right. Parameter defined as job parameter in LongevityPipeline. so it always will be set at least with default value ''. So if it is not configured , this if condition will be skipped. and old behavior will be used.
in case that this parameter will set other than integer, groovy will raise an exception and job will not be run. I don't see, that it is need to add any additional checks on groovy and do code more complex.

in case that these parameters will be set as negative integers, i will add check to sct_config validations